phpBB/language/en/common.php

```diff
@@ -357,6 +357,7 @@
 	'HIDE_ME'						=> 'Hide my online status this session',
 	'HOURS'							=> 'Hours',
 	'HOME'							=> 'Home',
+	'HTTP_HANDLER_NOT_FOUND'		=> 'The operation could not be completed because cURL PHP extension and allow_url_fopen PHP ini setting have been disabled and no other HTTP handler could be found.',
```

Comment:
```suggestion
	'HTTP_HANDLER_NOT_FOUND'		=> 'The operation could not be completed because the cURL PHP extension and allow_url_fopen PHP ini setting have been disabled and no other HTTP handler could be found.',
```